dbt/adapters/snowflake/connections.py

```diff
@@ -476,7 +476,7 @@ def execute(
         if fetch:
             table = self.get_result_from_cursor(cursor, limit)
         else:
-            table = dbt.common.clients.agate_helper.empty_table()
+            table = dbt_common.clients.agate_helper.empty_table()
```

Comment:
Is there a reason to not use the same pattern as other imports instead, ie

`from dbt_common.clients.agate_helper import empty_table`

and then this line is

`table = empty_table()`

Reply:
was actually looking at that myself earlier, going to double check that we aren't using something else called agate_helpper to prevent a name collision, that's really the only other use case I've seen it done this way via exceptions.